ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/scan/view/ScanView.kt

```diff
@@ -297,37 +316,43 @@ fun ScanCameraView(
                         ViewGroup.LayoutParams.MATCH_PARENT
                     )
                 }
+                previewView.contentDescription = contentDescription
                 val preview = androidx.camera.core.Preview.Builder().build()
                 val selector = CameraSelector.Builder()
                     .requireLensFacing(CameraSelector.LENS_FACING_BACK)
                     .build()
                 preview.setSurfaceProvider(previewView.surfaceProvider)
 
+                val imageAnalysis = ImageAnalysis.Builder()
+                    .setBackpressureStrategy(ImageAnalysis.STRATEGY_KEEP_ONLY_LATEST)
+                    .build()
+
+                imageAnalysis.setAnalyzer(
```

Comment:
This is another example of the type of callback/lifecycle thing that could potentially get triggered after the view is being torn down.

Likewise I want to poke at this.  This one will be trickier since it is an Android view and not inside Compose.